src/core/Akka/Serialization/Serializer.cs

```diff
@@ -121,7 +121,17 @@ public static int GetSerializerIdentifierFromConfig(Type type, ExtendedActorSyst
         {
```

Comment:
cc @Horusiath what does this method actually do?
Each serializer does have an identifier on the serializer, why is this needed?
Can we not just use ` Sys.Serialization.FindSerializerFor(obj).Identifier`
Does the identifier need to be configurable?